src/LoRa.cpp

```diff
@@ -607,6 +607,32 @@ void LoRaClass::setOCP(uint8_t mA)
   writeRegister(REG_OCP, 0x20 | (0x1F & ocpTrim));
 }
 
+void LoRaClass::setGain(uint8_t gain)
+{
+  // check allowed range
+  if(gain > 6) {
```

Comment:
Nitpick: please add an space after the `if` for consistency like the current contents of the file.

Reply:
Ok, I will fix it today.

Reply:
Done at https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/42565206b85e2e43ed61151c8f192cdc073fbc9b